File: CSharp/Struct/ToString.cs
using static System.Console;

public class Program {
	public static void Main() => WriteLine(new Cores(80, 20, 160));
}

struct Cores {
	public byte R { get; }
	public byte G { get; }
	public byte B { get; }
	
	public Cores(byte r, byte g, byte b) {
		R = r;
		G = g;
		B = b;
	}
	public override string ToString() => $"{R}, {G}, {B}";
}

//https://pt.stackoverflow.com/q/236022/101

File: Java/Method/ToString.java
public String toString() {
     return String.format("O valor do objeto é %s", String.valueOf(value);
}

public class comp {
    private String nome;
    private Data aniversario;
    public comp (String n, Data a) {
        nome = n;
        aniversario = a;
    }
    public String toString() {
        return String.format("%s - %s", nome, aniversario);
    }
    public String getTextual() {
        return String.format("O teu nome é %s , e a fazes anos no dia %s", nome, aniversario);
    }
}

//https://pt.stackoverflow.com/q/212754/101
